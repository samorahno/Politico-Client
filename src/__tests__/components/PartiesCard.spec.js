import React from 'react';
import { shallow } from 'enzyme';
import PartiesCard from '../../components/common/partiesCard';

describe('Parties Card Component', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<PartiesCard />);
        expect(component).toMatchSnapshot();
    });
});
