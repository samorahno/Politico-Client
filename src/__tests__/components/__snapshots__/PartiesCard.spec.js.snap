// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parties Card Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
